contracts/notification-contracts/src/main/java/org/java/mentorship/contracts/notification/client/NotificationFeignClient.java

```diff
@@ -0,0 +1,59 @@
+package org.java.mentorship.contracts.notification.client;
+
+import org.java.mentorship.contracts.notification.dto.Notification;
+import org.java.mentorship.contracts.notification.dto.NotificationChannel;
+import org.java.mentorship.contracts.notification.dto.NotificationType;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.Map;
+
+@FeignClient(value = "notificationClient",
+        url = "${service.notification.url}")
+public interface NotificationFeignClient {
+
+    @RequestMapping(
+            method = RequestMethod.GET,
+            value = "/notifications"
+    )
+    List<Notification> getNotifications(@RequestParam(required = false, name = "id") Integer id,
+                                        @RequestParam(required = false, name = "userId") Integer user_id,
+                                        @RequestParam(required = false, name = "email") String email,
+                                        @RequestParam(required = false, name = "payload") Map<String, String> payload,
```

Comment:
I doubt we'll ever use all of these parameters

Notifications should be retrievable by:
userId (please make it camelCase)
email
channel (just one)
type (just one)
marked